src/webgl/p5.Camera.js

```diff
@@ -488,7 +505,7 @@ p5.Camera = class Camera {
     this._renderer = renderer;
 
     this.cameraType = 'default';
-
+    this.useLinePerspective = true;
```

Comment:
Do you think calling `perspective()` should set this to true and calling `ortho()` or `frustum()` should set this to false? A user could then override it with `linePerspective()`, but still match the defaults we have now.

Reply:
Yes maybe, I think  it makes sense to associate the behavior of linePerspective with the type of camera projection being used. If you're using `perspective()`, which simulates realistic perspective, having linePerspective set to true by default aligns well with that mode. Similarly, when using `ortho()` or `frustum()`, which do not simulate realistic perspective, setting linePerspective to false by default is appropriate.
This way, the default behavior aligns with the common expectations of how lines should behave based on the chosen camera projection. 
let me know what's your take on this.
Thanks!

Reply:
I think that makes sense! we can mention in the documentation for `linePerspective` that `perspective()`, `ortho()`, and `frustum()` all change the value of `linePerspective`, so you can call it after those calls to override it.